person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
+  private :of_age?
+  def can_use_services?
+    of_age? || @parent_permission == true
```

Comment:
- [ ] **Required** This is required to be a public method

![image](https://user-images.githubusercontent.com/27709832/212885027-3b0bf716-92a8-4d9a-ae36-91012e74205e.png)

- [ ] **Optional:** The attribute @parent_permission is set default to true. You could just use it since it's already true. It's like saying `true == true`.
```
def can_use_services?
    of_age? || @parent_permission
 end
```